### DISCUSSION TITLE
Enabling/disabling specific exporters doesn't work

### DISCUSSION THREAD

**Discussion by mat-rumian** [open-telemetry/opentelemetry-java-instrumentation#2282] [Q&A] [answered]
In previous releases (for sure in *0.14.0*) I was able to setup only traces exporter (oTLPgRPCSpanExporter) by setting an environment variable: `OTEL_EXPORTER=otlp_span`. In current release (*0.16.1*) I was trying do the same by setting `OTEL_TRACES_EXPORTER=otlp` and `OTEL_METRICS_EXPORTER=none` but looks like it doesn't work.

An error in the logs:

```
[opentelemetry.auto.trace 2021-02-14 21:04:35:062 +0000] [grpc-default-executor-6] WARN io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter - Failed to export metrics
io.grpc.StatusRuntimeException: UNIMPLEMENTED: unknown service opentelemetry.proto.collector.metrics.v1.MetricsService
```

**Comment by mateuszrzeszutek** [comment, marked as answer]
Have you tried using the system property instead (`-Dotel.metrics.exporter=none`)?

**Reply by mat-rumian** [reply]
Thanks it works fine, I also checked again environment variables and it came out I made a mistake... I'm sorry. Functionality works fine for both - property and env var.

**Reply by mateuszrzeszutek** [reply]
Good to hear that - I started worrying that there's some difference in behavior between env vars & system props 😄

**Reply by hballangan-mdsol** [reply]
Hi! Related to this issue, OTEL_METRICS_EXPORTER is working for 16.1 and has been clarified as a mistake but OTEL_TRACES_EXPORTER=none is not recognized. Only OTEL_METRICS_EXPORTER=none is working for me.
